node/test/PublicAPITest.ts

```diff
@@ -7,14 +7,76 @@ import { assert } from 'chai';
 import * as SignalClient from '../index';
 
 describe('SignalClient', () => {
-  it('can generate and serialize PrivateKeys', () => {
-    const a = new SignalClient.PrivateKey();
-    const b = new SignalClient.PrivateKey();
-    assert.equal(a.serialize().length, 32, 'correct length');
-    assert(a.serialize().equals(a.serialize()), 'repeatable');
+  it('ECC signatures work', () => {
+    const priv_a = SignalClient.PrivateKey.generate();
+    const priv_b = SignalClient.PrivateKey.generate();
+    assert.equal(priv_a.serialize().length, 32, 'correct length');
+    assert(priv_a.serialize().equals(priv_a.serialize()), 'repeatable');
```

Comment:
If you wish:

```suggestion
    assert.deepEqual(priv_a.serialize(), priv_a.serialize(), 'repeatable');
```

You can do this any time you're comparing buffers. It can give a clearer error message, showing you the diff instead of "assertion failed".

You can use this elsewhere in this file, too.

Reply:
Nice thanks!

[You are looking at literally the first JS/TS I have ever written so any suggestions on style, easier ways of doing things etc are super useful]

Reply:
This is a _lot_ better than the first JS I wrote...and not far off the JS I write today!